_episodes/02-first-example.md

```diff
@@ -43,6 +43,8 @@ $ cd reana-demo-root6-roofit
 ~~~
 {: .bash}
 
+For those who are using a remote setup, first go down to the "Connect REANA client to remote REANA cluster" section to set up `reana-client`. Then return back to this section.
```

Comment:
The lesson asks the users to do some `reana-client` commands. Would you like me to move this sentence down? I thought it would be good to get the setup out of the way, since it already needs to be done.

Reply:
This section is meant to demonstrate/motivate the general reana workflow. Probably that's why it's before the installation/setup. Students are only mean to clone the repo and look at the files there in this section. 

Installing reana and running things comes after. I think this makes sense.

Reply:
Okay, I'll add it to setup section instead.